src/clj_async_profiler/wwws.clj

```diff
@@ -85,6 +85,10 @@
     (.setExecutor nil)
     (.start)))
 
+(defn get-port [^HttpServer server]
+  (some-> server
+          (-> .getAddress .getPort)))
```

Comment:
`some->` can thread through multiple forms, you don't need `->` here.

Reply:
Yep, but if you only use `some->` it will do a nil check on every step whereas the one I used only does it where I think it's needed. That being said I can change if you prefer it that way.

Reply:
Won't hurt!